rules-reviewed/quarkus/springboot/springboot-di-to-quarkus.windup.xml

```diff
@@ -19,7 +19,7 @@
                 <dependency groupId="org.springframework" artifactId="spring-beans"/>
             </when>
             <perform>
-                <hint title="Replace the SpringBoot Dependency Injection artifact with Quarkus 'spring-di' extension" effort="1" category-id="mandatory">
+                <hint title="Replace the SpringBoot Dependency Injection artifact with Quarkus 'spring-di' extension" effort="1" category-id="potential">
```

Comment:
Why has this changed to potential? I thought if you didn't do this it wouldn't work?

Reply:
You're right and wrong.
The application must have it as a lib when running but this doesn't mean to have to explicitly declare it as a dependency in the pom.xml.
This artifact will be available also if you add the `quarkus-spring-web` to the pom.xml without having to add also this `quarkus-spring-di` dependency.
So if the user changes the mandatory `quarkus-spring-web` this issue will be fixed "for free".
If not, then it has to be added.
This "maybe situation" made it potential to me because "_there is not enough detailed information to determine if the task is mandatory for the migration to succeed_"